apps/cli/src/commands/run.ts

```diff
@@ -63,18 +73,27 @@ export default class Run extends Command {
             WS_URL: "ws://anvil:8545",
         };
 
-        // resolve compose file location based on version
-        const composeFile = path.join(
+        // Construct the path for Docker Compose files
+        const composeFileDev = path.join(
             path.dirname(new URL(import.meta.url).pathname),
             "..",
             "node",
             "docker-compose-dev.yml"
         );
+        const composeFileHost = path.join(
+            path.dirname(new URL(import.meta.url).pathname),
+            "..",
+            "node",
+            "docker-compose-host.yml"
+        );
+
+        // Use composeFileDev by default, add composeFileHost if --no-backend is defined
+        const selectedComposeFile = flags["no-backend"] ? [composeFileDev, composeFileHost] : composeFileDev;
 
         const args = [
             "compose",
             "--file",
-            composeFile,
+            selectedComposeFile,
```

Comment:
we need a new `--file` for each file, like:

```
docker compose --file docker-compose-dev.yml --file docker-compose-host.yml ...
```

Reply:
Actually the documentation says a CSV is accepted.

But I think I prefer a profile definition

Reply:
CSV didn't worked for me :thinking:

Agree that profile is a cleaner solution.

Reply:
> But I think I prefer a profile definition

I think we can't use profile, because the only thing we need is to replace some ENVs and I think having an override compose file is better solution.